q2_composition/_ancombc.py

```diff
@@ -93,59 +93,66 @@ def _ancombc(table, metadata, formula, p_adj_method, prv_cut, lib_cut,
     for term in formula_terms:
         metadata.get_column(term)
 
+    # Pulling default reference levels based on formula terms
+    if reference_levels is None:
+        reference_levels = []
+        for term in formula_terms:
+            if isinstance(metadata.get_column(term),
+                          CategoricalMetadataColumn):
```

Comment:
Open to other variable names here - I used `term_alpha_value` because this is the value from the alphabetized column associated with that specific term, but alpha might also be an overloaded word in the QIIME 2 ecosystem.

Reply:
Term seems fine